src/EditorFeatures/TestUtilities/TestWaitIndicator.cs

```diff
@@ -5,27 +5,24 @@
 using System.Threading;
 using Microsoft.CodeAnalysis.Editor.Host;
 using Microsoft.CodeAnalysis.Shared.Utilities;
+using Microsoft.CodeAnalysis.Host.Mef;
 using VisualStudioIndicator = Microsoft.VisualStudio.Language.Intellisense.Utilities;
 
 namespace Microsoft.CodeAnalysis.Editor.UnitTests.Utilities
 {
     [Export(typeof(IWaitIndicator))]
     [Export(typeof(VisualStudioIndicator.IWaitIndicator))]
+    [PartNotDiscoverable]
     public sealed class TestWaitIndicator : IWaitIndicator, VisualStudioIndicator.IWaitIndicator
     {
-        public static readonly TestWaitIndicator Default = new TestWaitIndicator();
-
         private readonly IWaitContext _waitContext;
-        private readonly Microsoft.VisualStudio.Language.Intellisense.Utilities.IWaitContext _platformWaitContext = new UncancellableWaitContext();
-
-        public TestWaitIndicator()
-            : this(new UncancellableWaitContext())
-        {
-        }
+        private readonly VisualStudioIndicator.IWaitContext _platformWaitContext = new UncancellableWaitContext();
 
-        internal TestWaitIndicator(IWaitContext waitContext)
+        [ImportingConstructor]
+        [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
+        internal TestWaitIndicator([Import(AllowDefault = true)] IWaitContext waitContext)
```

Comment:
So we import our version, but the platform one is not that same one? This seems really funky. I guess this isn't new, I'm just not quite sure what it means.

Reply:
This change is now removed from this pull request.